content/docs/add-react-to-a-website.md

```diff
@@ -1,35 +1,34 @@
 ---
 id: add-react-to-a-website
-title: Add React to a Website
+title: React को Website से जोड़े
 permalink: docs/add-react-to-a-website.html
 redirect_from:
   - "docs/add-react-to-an-existing-app.html"
 prev: getting-started.html
 next: create-a-new-react-app.html
 ---
+अपनी आवश्यकता के अनुसार कम या ज्यादा React का उपयोग करें
 
-Use as little or as much React as you need.
+क्रमिक गोद या अपने तरीके से उपयोग करने के लिए शुरू से ही इसी तरह डिज़ाइन की गई है, और **आप आवश्यकतानुसार कम या ज्यादा इसका उपयोग कर सकते हैं**। शायद आप केवल एक मौजूदा पेज पर कुछ "अन्तरक्रियाशीलता के छिड़काव( इसके अंतर्गत कुछ छोटा सा काम )" जोड़ना चाहते हैं। React Component ऐसा करने का एक शानदार तरीका है।
```

Comment:
`Component` => `कौम्पोनॅन्ट`

check #2 for more commonly used words and its translation.
Also you can get more information on translation there

Reply:
yes bro